dbdev/models.py

```diff
@@ -0,0 +1,45 @@
+import psycopg2
+
+from flask import abort,Response
+
+
+connect = psycopg2.connect("dbname='ridemyway' host='localhost' user='dario' password='riot'")
+curs = connect.cursor()
+
+
+
+#Class to add new users to the db
+class DbManager(object):
+      
+    #class methods
+    def __init__(self):
```

Comment:
If this section of your code is not needed then I don't see the need of having it here. What are your thoughts on this?